components/cam/cime_config/config_component.xml

```diff
@@ -374,20 +377,25 @@
     <desc compset="CAM[45].*_BGC%B">CAM prognostic CO2 cycle turned on.</desc>
     <!-- Super-Parameterization -->
     <desc compset="_CAM5%SP1V1_"    >Super-parameterized E3SM with 64x1km CRM, RRTMG radiation, 1-moment microphysics and prescribed aerosol</desc>
+    <desc compset="_CAM5%SP1V1-MAML">Super-parameterized E3SM with 64x1km CRM, RRTMG radiation, 1-moment microphysics, prescribed aerosol and MAML </desc>
```

Comment:
Let's include a note that the time step defaults to 5 min (since this means it's somewhat specific to ne30 and needs to be reduced further for ne120) and that the reduced rad is disabled.